lib/Backend/GlobOptArrays.cpp

```diff
@@ -136,13 +145,74 @@ bool GlobOpt::ArraySrcOpt::CheckOpCode()
             needsLength = true;
             break;
 
+        case Js::OpCode::IsIn:
```

Comment:
>Js::OpCode::IsIn [](start = 13, length = 16)

Please add tests

Reply:
Is there a way to add test which will emit a trace to say that 'in' check is removed? That would be a good regression test (like bailout tests where we print that bailout happened)

---
In reply to: [174537610](https://github.com/Microsoft/ChakraCore/pull/4819#discussion_r174537610) [](ancestors = 174537610)

Reply:
-testtrace:BoundCheckElimination will trace that bound checks were eliminated, but not specify that it was the 'in' operator. -trace:BoundCheckElimination includes the instruction which contains the opcode but is also less stable because it includes symbol numbers. Neither specify that the check was removed. I'll add a stable trace for that.

Reply:
Added a unit test